src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -87,6 +89,87 @@
     }
     
     /**
+     * Generates the summary of the feedback sessions (in which at least one
+     * email has been sent to students) email for the given {@code courseId} for {@code student}
+     */
+    public EmailWrapper generateFeedbackSessionResendAllLinksEmail(String courseId,
+            StudentAttributes student) {
```

Comment:
Similar to the one of the previous comments, since this goes beyond all feedback session links (i.e course join link may also be involved), you may want to rename the method and also its header comment.

On a more minor note, `StudentAttributes student` should be fine on the previous line, but this may or may not be relevant once you rename the method.